docs/output-files/clst-transition-density-clusters.md

```diff
@@ -18,10 +18,103 @@ Transition density cluster files are ASCII files with the extension `.clst`. The
 1. TOC
 {:toc}
 
+
+---
+
 ## Description
 
+The transition density cluster files are written in ASCII format and contain methods, parameters and results of the TDP clustering.
+
+They are created in the `/transition_analysis/clustering` analysis sub-folder when completing a clustering procedure and exporting clusters from window 
+[Export options](../transition-analysis/functionalities/set-export-options.html) in module Transition analysis.
+
+
+---
+
 ## File name
 
+The file is named by the user during the export process.
+
+By default, the file is named after the selected <u>project file</u> and is appended with the extension `_[Ddd]`, where `[Ddd]` is the data type written in the file.
+
+Data types supported in transition density cluster files are:
+* `I[i]-[L]`: intensities in detection channel indexed `[i]` upon illumination with laser wavelength `[L]` nm
+* `FRET[D]to[A]`: FRET from donor emitter detected in channel indexed `[D]` to acceptor emitter detected in channel indexed `[A]`
+* `S[Eee]`: stoichiometry of emitter detected in channel labelled `[Eee]`
```

Comment:
For gamma correction yes, but the stoichiometry of any emitter can be calculated as long as intensities are collected upon each emitter-specific illumination. This can be used in molecule sorting for instance.